README.md

```diff
@@ -65,7 +74,7 @@ Blackholio/
 ---
 
 ### **Requirements**
-- **Unity**: Version `2022.3.32f1 LTS` or later.
+- **Unity**: Version `2021.2` or newer.
```

Comment:
We should use this same Unity version everywhere because this is our actual minimum spec `2021.2`